pkg/registry/constants.go

```diff
@@ -0,0 +1,19 @@
+//
+//
+// Copyright Red Hat
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package registry
+
+const maxTruncLength = 63
```

Comment:
Out of curiosity, what is the difference between the `pkg/registry/contstants.go` & `pkg/registry/defaults.go`?

Reply:
@thepetk Yes, `defaults.go` is everything related to default values in the `DevfileRegistry` CR whereas `constants.go` is just global constants such as the one we have for index server: https://github.com/devfile/registry-support/blob/b950a067ef2eb2a1bb08ea1658bedeb1b600e722/index/server/pkg/server/constants.go